docs.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Generates the How To section of the docs.
+ * Ensures the code for clasp commands is well documented.
+ * @todo Generate a list of commands for the beginning of the README.
+ */
+const fs = require('fs');
+const parseComments = require('parse-comments');
+const extract = require('extract-comments');
+const file = fs.readFileSync('./index.ts').toString();
+const ucfirst = require('ucfirst');
```

Comment:
Not a big deal, but we're importing a one-liner:

```
'use strict';

module.exports = function ucfirst(s) {
  return s.charAt(0).toUpperCase() + s.substr(1);
};
```
from [here](https://github.com/deathcap/ucfirst/blob/master/ucfirst.js)

Can we just do this ourselves?

Reply:
Sure, I'll update soon.